bluepyopt/ephys/protocols.py

```diff
@@ -270,15 +284,23 @@ def _reduce_method(meth):
                                        sim=sim)
         return responses
 
-    def instantiate(self, sim=None, icell=None):
+    def instantiate(self, sim=None, icell=None, LFPyCell=None):
         """Instantiate"""
 
         for stimulus in self.stimuli:
-            stimulus.instantiate(sim=sim, icell=icell)
+            if LFPyCell is not None:
```

Comment:
Again here, if LFPyCell would be part of icell all this logic was not necessary

Reply:
I beliebe `icell` is a NEURON object, so we can't add the `LFPy.Cell` object to it. @wvangeit can you confirm?

Reply:
Ah ok, sorry, you're right. I thought for a sec icell was the bpopt cell object

Reply:
But still, not all stimuli will have LFPyCell as argument, right? So this will break code that doesn't accept this argument

Reply:
Yes. Maybe a try except here? I'm skeptical about hardcoding types. For example, [here] (https://github.com/alejoe91/multimodalfitting/blob/master/multimodalfitting/ecode/ecodestimuli.py) we define externally many LFPy-compatible stimuli... alternatively, each stimulus could have a flag `supports_lfp=False` by defalut at class level and we can check that. What do you think?

Reply:
We could pass cell instead of icell to the instantiate method